lib/board.rb

```diff
@@ -0,0 +1,77 @@
+require 'pry'
+##
+# => This handles the behavior of the board
+##
+class Board
+  # => Sets the cells getter and setter
+  attr_accessor :cells
+
+  ##
+  # => This instantiates a new board with a fresh grid
+  ##
+  def initialize
+    reset!;
+  end
+
+  ##
+  # => Creates a fresh grid for a board
+  ##
+  def reset!
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
```

Comment:
Another way to instantiate an Array with `i` values is `Array.new(9, " ")` (see [Array.new](http://ruby-doc.org/core-2.2.0/Array.html#method-c-new)), where `9` is the number of elements in the array and `" "` is the default value for the elements. It's more concise and will prevent errors from repeatedly typing `" "`.